RecoLocalTracker/SiPixelClusterizer/plugins/PixelThresholdClusterizer.cc

```diff
@@ -119,6 +144,11 @@ void PixelThresholdClusterizer::clusterizeDetUnitT( const T & input,
   
   detid_ = input.detId();
   
+  // Set separate cluster threshold for L1 (needed for phase1)
+  auto clusterThreshold = theClusterThreshold;
+  int layer = (DetId(detid_).subdetId()==1) ? PXBDetId(detid_).layer() : 0;
```

Comment:
@jkarancs are you sure you can use the PXBDetId class for the phase 1 detector? Between phase 0 and phase 1 the detid definition has changed and you should use TrackerTopology.